docs/fsharp/tutorials/type-providers/generating-fsharp-types-from-edmx.md

```diff
@@ -133,10 +133,10 @@ In this step, you create and configure the type provider with the EDMX connectio
 <br />
 
 ```fsharp
-type edmx = EdmxFile<"Model1.edmx", ResolutionFolder = @"<path-tofolder-that-containsyour.edmx-file>>
+type edmx = EdmxFile<"Model1.edmx", ResolutionFolder = @"<path-tofolder-that-containsyour.edmx-file>">
 
 let edmConnectionString =
-getEDMConnectionString("Data Source=SERVER\instance;Initial Catalog=School;Integrated Security=true;")
+  getEDMConnectionString("Data Source=SERVER\instance;Initial Catalog=School;Integrated Security=true;")
```

Comment:
Similar to above, this is a connection, so I think the variable should be named `edmConnection`.

Also, `EntityConnection` is disposable, so shouldn't `use` be used here instead of `let`? (And probably for `context` below too.)